coreos-base/coreos-cryptagent/coreos-cryptagent-9999.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2018 CoreOS.
```

Comment:
I don't recall us referring to the company as just "CoreOS" elsewhere in copyright notices.  I'd suggest "CoreOS, Inc."

Out of curiosity, I decided to check what we use most regularly, and there are some odd ones in there.
```
grep --include='*.ebuild' -hir 'Copyright.*CoreOS' ../third_party/coreos-overlay | sort | uniq -c | sort -nr
     17 # Copyright (c) 2014 CoreOS, Inc.. All rights reserved.
      8 # Copyright (c) 2013 CoreOS, Inc.. All rights reserved.
      5 # Copyright 2016 CoreOS, Inc.
      5 # Copyright (c) 2017 CoreOS, Inc.. All rights reserved.
      5 # Copyright (c) 2015 CoreOS, Inc.. All rights reserved.
      4 # Copyright 2013 The CoreOS Authors
      3 # Copyright (c) 2017-2018 CoreOS, Inc.. All rights reserved.
      3 # Copyright (c) 2014 CoreOS, Inc. All rights reserved.
      3 # Copyright (c) 2013 CoreOS Inc. All rights reserved.
      2 # Copyright 2017 CoreOS, Inc.
      2 # Copyright 2016 CoreOS, Inc
      2 # Copyright 2014-2016 CoreOS, Inc.
      2 # Copyright 2014 CoreOS, Inc.
      2 # Copyright (c) 2016 CoreOS, Inc.. All rights reserved.
      2 # Copyright (c) 2015 CoreOS, Inc.
      2 # Copyright (c) 2013 The CoreOS Authors. All rights reserved.
      1 # Copyright 2017 The CoreOS Authors
      1 # Copyright 2015 CoreOS, Inc.
      1 # Copyright 2015 CoreOS, Inc
      1 # Copyright 2014 The CoreOS Authors
      1 # Copyright 2014 CoreOS, Inc
      1 # Copyright 1999-2013 CoreOS Inc
      1 # Copyright (c) 2016 CoreOS, Inc. All rights reserved.
      1 # Copyright (c) 2015 The CoreOS OS Authors. All rights reserved.
      1 # Copyright (c) 2015 CoreOS Inc.
      1 # Copyright (c) 2013 CoreOS Authors. All rights reserved.
      1 # Copyright (c) 2012 The CoreOS Authors. All rights reserved.
```